pkg/verifier/notation/notation.go

```diff
@@ -175,11 +175,13 @@ func (v *notationPluginVerifier) Verify(ctx context.Context,
 	}
 
 	return verifier.VerifierResult{
-		Name:       v.name,
-		Type:       v.verifierType,
-		IsSuccess:  true,
-		Message:    "signature verification success",
-		Extensions: extensions,
+		Name:         v.name,         // Deprecating Name in v2, switch to VerifierName instead.
+		Type:         v.verifierType, // Deprecating Type in v2, switch to VerifierType instead.
+		VerifierName: v.name,
+		VerifierType: v.verifierType,
+		IsSuccess:    true,
+		Message:      "signature verification success",
```

Comment:
i noticed the cosign message is "cosign verification failed". Should notation be consistent?

Reply:
Good call. Actually I plan to remove the verifier name from the message since we already have the `verifierName` field. So the cosign message will look like: signature verification failed.